.travis.yml

```diff
@@ -1,18 +1,16 @@
 dist: trusty
 language: node_js
+os:
+- linux
+- windows
 node_js:
-- '4'
 - '6'
 - '8'
+- '10'
 addons:
   apt:
```

Comment:
```suggestion
  chrome: stable
```

Reply:
since puppeteer is used for browser tests, no need of chrome addons.

Reply:
@pavanteja-potnuru Yes, this is more of an experiment. We'll end up using one of the two.